packages/appkit/src/client.ts

```diff
@@ -570,7 +570,12 @@ export class AppKit {
     connectedWalletInfo,
     chain
   ) => {
-    AccountController.setConnectedWalletInfo(connectedWalletInfo, chain)
+    AccountController.setConnectedWalletInfo(
+      connectedWalletInfo
+        ? { type: ProviderUtil.getProviderId(chain), ...connectedWalletInfo }
```

Comment:
If somehow `connectedWalletInfo` have `type`, it'll override the `ProviderUtil.getProviderId(chain)`. Is that expected? If not let's do; 
```suggestion
        ? { ...connectedWalletInfo, type: ProviderUtil.getProviderId(chain) }
```

instead

Reply:
In my pov this is fine, allowing to enforce a type if needed and letting a default value in case not provided. But we may change if you think that would be better.